server/embed/config_test.go

```diff
@@ -68,16 +73,18 @@ func TestConfigFileOtherFields(t *testing.T) {
 		t.Fatal(err)
 	}
 
-	if !cfg.ForceNewCluster {
-		t.Errorf("ForceNewCluster = %v, want %v", cfg.ForceNewCluster, true)
-	}
-
 	if !ctls.equals(&cfg.ClientTLSInfo) {
 		t.Errorf("ClientTLS = %v, want %v", cfg.ClientTLSInfo, ctls)
 	}
 	if !ptls.equals(&cfg.PeerTLSInfo) {
 		t.Errorf("PeerTLS = %v, want %v", cfg.PeerTLSInfo, ptls)
 	}
+
+	testutil.AssertEqual(t, false, cfg.ForceNewCluster, "ForceNewCluster does not match")
```

Comment:
@ptabor I noticed we have this test helper function so I used that, let me know if you meant something different? Thank you! (Also adjusted for the forcenewcluster, hope that is okay?)

Reply:
In the other tests we use "https://github.com/stretchr/testify/assert" that is pretty mature and convenient.

Reply:
Will do that I like that project! I did a quick search so that internal package showed up, is it okay if I open a tech debt issue so we conform to https://github.com/stretchr/testify/assert everywhere?